up-linux-streamer/DEFAULT_CONFIG.json5

```diff
@@ -22,7 +22,7 @@
     },
     zenoh_transport_config: {
       // Endpoint to be used when starting Zenoh session
```

Comment:
Should change comment to describe what it is currently

Reply:
done